lib/Basics/ArangoGlobalContext.cpp

```diff
@@ -98,6 +98,23 @@ ArangoGlobalContext::ArangoGlobalContext(int /*argc*/, char* argv[],
 #endif
 #endif
 
+  bool lastOptionName{false};
```

Comment:
What is this magic for? What happens if the database directory is not given on the command line? Don't we have a DatabasePathFeature for this?

Reply:
I also do not understand the logic with `lastOptionName`.

Reply:
I think this is buggy.

Reply:
At the very least it deserves an explanation.